packages/cli/src/utils/environment-tasks.ts

```diff
@@ -98,19 +98,29 @@ export const start: ListrTask<IStart> = {
     {
       title: 'Starting the Engine',
       skip: async ctx => await isRunning(ctx.endpoint) || (await listContainers({ label: [engineLabel] })).length > 0,
-      task: ctx => {
-        write(ctx.configDir, {
-          account: {
-            mnemonic: ctx.mnemonic
-          }
-        })
-        return createContainer(`${image}:${ctx.version}`, ctx.configDir)
-      }
+      task: ctx => createContainer(`${image}:${ctx.version}`, ctx.configDir)
     },
     {
       title: 'Waiting for the Engine to be ready',
       skip: ctx => isRunning(ctx.endpoint),
       task: async ctx => waitToBeReady(ctx.endpoint)
+    },
+    {
+      title: 'Ensure balance',
+      task: async ctx => {
+        const api = new API(ctx.endpoint)
+        const engineAccount = await api.account.import(ctx.config.engine.account.mnemonic)
+        ctx.engineAddress = engineAccount.address
+        const userAccount = await api.account.import(ctx.config.mnemonic)
+        const userBalance = userAccount.coins.find(x => x.denom === 'atto')
+        if (userBalance && parseInt(userBalance.amount, 10) > 1000) return
+        const transferMsg = api.account.transferMsg(engineAccount.address, userAccount.address, [{
+          amount: 1e18.toString(),
```

Comment:
is it enough? 1e18 is 1 actual coin.
in the engine e2e, it transfers 100,000,000 x 10^18 atto

Reply:
Can definitely be increased, just here the engine will do all the processing, the CLI should just use that to deploy. So the question is is it enough to deploy multiple processes and services.
I think 100k is really a lot just for deployment